website/src/views/components/KeyboardShortcuts.tsx

```diff
@@ -33,8 +33,7 @@ const KeyboardShortcuts: React.FC = () => {
   const [helpShown, setHelpShown] = useState(false);
   const closeModal = useCallback(() => setHelpShown(false), []);
 
-  const mode = useSelector(({ settings }: State) => settings.mode);
-  const themeId = useSelector(({ theme }: State) => theme.id);
+  const store = useStore();
```

Comment:
Is it possible to use this instead so that we don't have to keep casting the store type?

```suggestion
  const store = useStore<State>();
```

Reply:
Woops, forgot that this exists. Fixed in https://github.com/nusmodifications/nusmods/pull/3165/commits/c282a2d31fe03787f1ee2bf9544d1c2453a29673.

Reply:
Thank you!